DOC-2597: Provide demo for Image Optimizer Signed Uploads option (#3572)
tinymce/tinymce-docs: modules/ROOT/partials/configuration/uploadcare_signed_upload_auth_provider.adoc

  plugins: 'uploadcare',
  toolbar: 'uploadcare',
  uploadcare_public_key: '<your-public-key>',
  uploadcare_signed_upload_auth_provider: (publicKey) => Promise.resolve({
    signature: 'sig',
    expire: 123
  }),
});
----

[TIP]
.Starter Project
====

> To get started quickly and understand how Uploadcare Signed Uploads integrates with {productname} {pluginname}, {companyname} recommends trying the starter project on GitHub to set up {pluginname} with Signed Uploads. Just clone the repository and follow the steps to set up the project. The link will take you to the GitHub page for the starter project, where the source code is available for you to view: link:https://github.com/tinymce/tinymce-uploadcare-signed-uploads-starter[{productname} {pluginname} Signed Uploads Starter Project].

Reviewer: ```suggestion
To get started quickly and understand how Uploadcare Signed Uploads integrates with {productname} {pluginname}, {companyname} recommends trying the starter project on GitHub to set up {pluginname} with Signed Uploads. Clone the repository and follow the steps to set up the project. The link leads to the GitHub page for the starter project, where the source code is available for viewing: link:https://github.com/tinymce/tinymce-uploadcare-signed-uploads-starter[{productname} {pluginname} Signed Uploads Starter Project].
```
